libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -27,6 +27,13 @@ import { BaseProjectShrinkwrapFile } from '../base/BaseProjectShrinkwrapFile';
  * This class implements common logic between "rush install" and "rush update".
  */
 export class WorkspaceInstallManager extends BaseInstallManager {
+  // Example: "C:\MyRepo\common\temp\npm-local\node_modules\.bin\npm"
+  private _packageManagerFilename: string = this.rushConfiguration.packageManagerToolFilename;
```

Comment:
I would prefer to set private variables that aren't static values/that reference `this` in the constructor (same goes for `_packageManagerEnv`), or just use the variable directly by referencing `this.rushConfiguration.packageManagerToolFilename`